.github/workflows/parsegithubissue.yaml

```
@@ -76,34 +90,65 @@ jobs:
             ${{ env.ISSUE_COMMENT }}
             Please edit this issue and select "Update comment" to resubmit.
 
-      - name: Remove labels
-        uses: actions-ecosystem/action-remove-labels@v1
-        if: env.PROCESS_SUCCESS == 'false'
+      - name: 'Az CLI login'
+        uses: azure/login@v1
         with:
-          labels: |
-            Processing
-            Success
-            
-      - name: Add error label
-        if: env.PROCESS_SUCCESS == 'false'
+          client-id: 2b6fa9d7-7dba-4600-a58a-5e25554997aa # DTS AKS Auto-Shutdown
+          tenant-id: 531ff96d-0ae9-462a-8d2d-bec7c0b42082 # HMCTS.NET
+          allow-no-subscriptions: true
+      - name: get aks resource details
+        if: env.PROCESS_SUCCESS == 'true'
+        continue-on-error: true
+        run: ./scripts/aks-resource-details.sh
+
+      - name: process costs
+        if: env.PROCESS_SUCCESS == 'true'
+        continue-on-error: true
+        run: |
+          python cost-calculator.py
```

maybe make this more resilient by not failing for cost calculation?
```suggestion
          python cost-calculator.py || echo "Calculating cost failed"
```